core/object/object.h

```diff
@@ -198,7 +198,7 @@ Array convert_property_list(const List<PropertyInfo> *p_list);
 struct MethodInfo {
 	String name;
 	PropertyInfo return_val;
-	uint32_t flags; // NOLINT - prevent clang-tidy to assign method_bind.h constant here, it should stay in .cpp.
+	uint32_t flags = 0;
```

Comment:
This is set to `METHOD_FLAG_NORMAL` in the constructors but `method_bind.h` shouldn't be included in `object.h`. Previously I had kept it uninitialized with a comment, but I figure we can just init to 0 and let the constructors set it to `METHOD_FLAG_NORMAL`.